### DISCUSSION TITLE
[Q] has getRows been removed from the build somehow because I can't find it on a worksheet via npm?

### DISCUSSION THREAD

**Discussion by nicferrier** [exceljs/exceljs#2415] [Q&A]
I am using version 4.1.1 with nodejs 12 and I'm getting `getRows is not a function`.

I can't find a mention of `getRows` in the dist files or in the lib/doc/worksheet file (in my package).

But obviously it does exist in the repository [here](https://github.com/exceljs/exceljs/blob/master/lib/doc/worksheet.js#L343)

The only thing I can think of is that it's been removed in some kind of build step?

**Comment by L-U-C-K-Y** [comment]
Just tried to use the `.getRows` function and it is also not working for me.

What I'll do now is use `.getRow` within a loop and commit to the stream.
